### PR TITLE
Struther no ville

### PR DESCRIPTION
**Pull request by struther**
@JasonMick How about this list?

Closes issue #8

### CODE REVIEW

File: struther-noVille.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+### Where to eat
+
+- Rebecca's
+- Deadwood
+- Fred's Ice Cream???
```
Comment (kwill307) [inline struther-noVille.md]: Maybe La Pita?

File: struther-noVille.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+### Where to eat
+
+- Rebecca's
```
Comment (kwill307) [inline struther-noVille.md:4]: What about Ollie's?

**Review by kwill307 — COMMENTED** [review]
Awesome list, lets chose one for tomorrow! @struther

**Comment by e1fabre** [comment]
where can I get some good burgers?

**Comment by mohammadkalash** [comment]
Famous Hamburger is a good choice. 👍

**Comment by JasonMick** [comment]
[jmick4] +2 @struther

File: struther-noVille.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+ - good breakfast and lunch
```
Comment (JasonMick) [inline struther-noVille.md:5]: [jmick4] Let's delete lunch, breakfast is the best meal of the day. :egg:

Comment (struther) [reply struther-noVille.md:5]: [struther] @JasonMick breakfast all day

File: struther-noVille.md (hunk 2 above)
Comment (struther) [reply struther-noVille.md:4]: @kwill307 
done
:beers:

**Comment by githubteacher** [comment]
img

File: struther-noVille.md (hunk 1 above)
Comment (struther) [reply struther-noVille.md]: done x2

**Pull request merged**